cli/pcluster/resources/batch/docker/scripts/entrypoint.sh

```diff
@@ -3,12 +3,20 @@ set -e
 
 echo "Starting Job ${AWS_BATCH_JOB_ID}"
 
-# mount nfs
-/parallelcluster/bin/mount_nfs.sh "${MASTER_IP}" "${SHARED_DIR}"
+ebs_shared_dirs=$(echo ${SHARED_DIR} | tr "," " ")
+
+for ebs_shared_dir in ${ebs_shared_dirs}
+do
+  # mount nfs
+  /parallelcluster/bin/mount_nfs.sh "${MASTER_IP}" "${ebs_shared_dir}"
+done
+
+arr=($ebs_shared_dirs)
+shared_dir=${arr[0]}
```

Comment:
I would avoid to have 2 variables with the same name and different capitalization. It might generate confusion and make it easier to introduce bugs.